### ISSUE TITLE
Obj definition

### ISSUE THREAD

**Issue by downFast** [owid/covid-19-data#44]
Hi
thanks for the work, could you help on defining your keys?

An example:

   "iso_code": "ITA",
   "location": "Italy",
   "date": "2020-01-05",
   "total_cases": 0,
   "new_cases": 0,
   "total_deaths": 0,
   "new_deaths": 0,
   "total_cases_per_million": 0,
   "new_cases_per_million": 0,
   "total_deaths_per_million": 0,
   "new_deaths_per_million": 0,
   "total_tests": null,
   "new_tests": null,
   "total_tests_per_thousand": null,
   "new_tests_per_thousand": null,
   "tests_units": "",
   "population": 60461828,
   "population_density": 205.859,
   "median_age": 47.9,
   "aged_65_older": 23.021,
   "aged_70_older": 16.24,
   "gdp_per_capita": 35220.084,
   "extreme_poverty": 2,
   "cvd_death_rate": 113.15100000000001,
   "diabetes_prevalence": 4.78,
   "female_smokers": 19.8,
   "male_smokers": 27.8,
   "handwashing_facilities": null,
   "hospital_beds_per_100k": 3.18

What do the following means?

tests_units

"extreme_poverty": 2, (what is 2?)

"handwashing_facilities": null, What is it? What do you mean?

"hospital_beds_per_100k": 3.18 This can be dangerous data if we don't say what beds for what hospital division

"gdp_per_capita": 35220.084, what is the value, million, thousand?

"cvd_death_rate": 113.15100000000001, what value is this rate?

Basically we'd need some definition on all the keys to better understand and work with this data

Thanks a lot

**Comment by edomt** [comment]
Hi @downFast 

A full codebook is made available ([owid-covid-data-codebook.md](https://github.com/owid/covid-19-data/blob/master/public/data/owid-covid-data-codebook.md)), with a description and source for each variable in the dataset.

As for "null", I assume that this value is added by the software you're using to download the data. Those cells are empty in our dataset, which means we don't have a value for that location/date/variable.

Edouard

**Comment by downFast** [comment]
Oh ok, missed that. Thanks

in regards of null, I meant what handwashing_facilities was but I guess it's in the codebook, null is what I get once i convert csv to json.

Any plan to add long and lat for each location btw?

thanks

**Comment by edomt** [comment]
@downFast We don't have plans to add lat/lon for now, but you can probably merge our dataset with [this one](https://gist.github.com/tadast/8827699) for example.

**Comment by downFast** [comment]
> this one

yes exactly what I wasl looking at. Thanks

**Comment by downFast** [comment]
Hi
I am wondering about some of your data:

extreme_poverty shows 2.0 for Italy, is it in % and then it means the 2%? 

And are we sure is correct?

Look at this https://www.istat.it/it/files//2011/07/poverta_15_7_2011_inglese.pdf

Also you say, for Italy: recovered: 0 on 9/5/20, but actually there are 103.031 recovered, data is provided here https://github.com/pcm-dpc/COVID-19/tree/master/dati-json

I mean, how riable is your data? I appreciate your work and I would like to implement it but I need to be sure of the data.

Thank you



--

**Comment by edomt** [comment]
Hi @downFast 

- Regarding extreme poverty, yes, 2.0 means 2%. I don't see any contradiction between the link you pointed to and our data. Note that the variable in the dataset is _extreme_ poverty. As noted in the codebook, the data on extreme poverty comes directly from the World Bank. You can find more information on this metric [on our website](https://ourworldindata.org/grapher/share-of-the-population-living-in-extreme-poverty?tab=chart&country=ITA), including a very detailed explanation of the "extreme poverty" measure in the "Sources" tab of the graph.

- We don't have any data on recovered people, so I'm not sure where you're getting this "0" from. Could you please point to the specific file and row?

Edouard

**Comment by downFast** [comment]
> * on our website

Ouch, my apologies, I was merging more data into it and got confused about recovered.

My fault :)

In terms of extreme_poverty, thanks a lot, I have read the source. Just that I know if you don't mind, how would you say the link I've posted doesn't contradicts your data?

I'm not arguing, I'm genuinely asking as i'd liek to better understand it, as I'd like to try to explain in plain words some of thoese keys, but I first need to understand it myself. So how would I check it against the link I've sent?

Thanks a lot

**Comment by downFast** [comment]
> Hi @downFast
> 
> * Regarding extreme poverty, yes, 2.0 means 2%. I don't see any contradiction between the link you pointed to and our data. Note that the variable in the dataset is _extreme_ poverty. As noted in the codebook, the data on extreme poverty comes directly from the World Bank. You can find more information on this metric [on our website](https://ourworldindata.org/grapher/share-of-the-population-living-in-extreme-poverty?tab=chart&country=ITA), including a very detailed explanation of the "extreme poverty" measure in the "Sources" tab of the graph.
> * Our dataset doesn't include any data on COVID-19 recoveries, so I'm not sure where you're getting this "0" from. Could you please point to the specific file and row?
> 
> Edouard

Ouch, my apologies, I was merging more data into it and got confused about recovered.

My fault :)

In terms of extreme_poverty, thanks a lot, I have read the source. Just that I know if you don't mind, how would you say the link I've posted doesn't contradicts your data?

I'm not arguing, I'm genuinely asking as i'd liek to better understand it, as I'd like to try to explain in plain words some of thoese keys, but I first need to understand it myself. So how would I check it against the link I've sent?

Thanks a lot

**Comment by downFast** [comment]
I mean where should I look at to compare data regarding the poverty level?

**Comment by edomt** [comment]
I don't have good knowledge of poverty levels in Italy, but given this sentence in the document:
_"In 2010 the relative poverty incidence was equal to 11%, whereas the absolute poverty to 4.6%."_
it seems plausible that the level of **extreme** poverty would be around 2%.
